### PR TITLE
Update crawling.md

### PR DESCRIPTION
**Pull request by liashahnazaryan**
# Description

Creating an article on `Crawling`

## Type of PR

- Creates the article _[Crawling]_

### Checklist:

- [x] I have read the [contributing guidelines](contributing.md).
- [x] I have followed the [style guide](http://machinetranslate.org/style).

### CODE REVIEW

File: customisation/crawling.md [hunk 1]
```diff
@@ -1,6 +1,27 @@
 ---
 parent: Customisation
-layout: coming_soon
 title: Crawling
 description: Crawling the web for machine translation training data
 ---
+
+**Crawling** or **web scraping** is the process of automatically collecting large amounts of [training data](/customisation/training-data.md) for [language models](/concepts/language-model.md) and machine translation systems.
+Web crawling is used to acquire [parallel data](/customisation/parallel-data.md) by visiting bilingual or multilingual websites and extracting parallel text sets.
+Another approach is to use crawling to collect monolingual text in a target language.
```
Comment (cefoo) [inline customisation/crawling.md:9]: Perhaps, it's simpler to say:

---

### Approaches

Bilingual or multilingual websites provide text sets for [parallel data](/customisation/parallel-data.md) extraction.
Web crawling also collects monolingual text in the target language.

File: customisation/crawling.md [hunk 2]
```diff
@@ -1,6 +1,27 @@
...
+
+### Process
+
+1. Visiting the first URL, manually specified by the user or selected from a pre-defined list of URLs
+2. Extracting URLs from the visited web page and adding them to a queue of URLs to be visited
+3. Extracting text data from the visited web page and storing it in a database for later use
+4. URL filtering to determine which URLs to visit next based on the pre-defined criteria
+5. Repetition of URL extraction, text extraction, and URL filtering for each URL in the queue
+6. Preprocessing and [filtering](/customisation/filtering.md) of the extracted text data to remove noise and normalise quality
```
Comment (cefoo) [inline customisation/crawling.md:18]: Let's use the infinitive for processes, as we did (you did, haha) in the [Attention](https://machinetranslate.org/attention#process) article.

**Review by cefoo — COMMENTED** [review]
Hi dear @liashahnazaryan!

Thank you so much for your contribution!

I've only added a couple of comments about formatting. 😊

File: customisation/crawling.md (hunk 1 above)
Comment (liashahnazaryan) [reply customisation/crawling.md:9]: Now that I think about this part, the word approach isn't relevant here as collecting both monolingual and parallel text sets are just two types of data that can be web-crawled. 
There are different approaches to crawling, like focused, incremental, etc., mentioned in some papers. But depending on the algorithm, the crawler can do all the things described in different approaches, which is why I didn't find it necessary to mention those.

Maybe just change it to (same as you proposed but without a heading): 
>**Crawling** or **web scraping** is the process of automatically collecting large amounts of [training data](/customisation/training-data.md) for [language models](/concepts/language-model.md) and machine translation systems.
Bilingual or multilingual websites provide text sets for [parallel data](https://github.com/customisation/parallel-data.md) extraction.
Web crawling also collects monolingual text in the target language.

Comment (cefoo) [reply customisation/crawling.md:9]: Yes, great!

**Comment by cefoo** [comment]
Fixes #72

**Pull request merged**